app/lib/backend_api_logic/service_extension.rb

```diff
@@ -22,7 +22,7 @@ def find_or_create_first_backend_api!
     end
 
     def act_as_product?
-      act_as_product && account&.provider_can_use?(:api_as_product)
+      provider_can_use?(:api_as_product)
```

Comment:
`account&.provider_can_use?(:api_as_product)`
Because this module belong to the service, not to the account.

Reply:
Spoke with Hery, I showed him that the test `test/unit/proxy_test.rb:74` fails without the `account&.` but it passes with it:
![image](https://user-images.githubusercontent.com/11318903/64326086-982d6a00-cfc9-11e9-9b14-e78270c44029.png)
![image](https://user-images.githubusercontent.com/11318903/64326105-a085a500-cfc9-11e9-83ec-a236aed4cfa3.png)

He told me that service also implements it:
https://github.com/3scale/porta/blob/c4896133d3e1d53f96cee0bdae80dbb00d80a482/app/lib/logic/rolling_updates.rb#L349-L355

So either the tests is wrong or it should be fixed in the module.
He will fix it 👍 💪

Reply:
Done, good job: 👍 💪 🥇 🕺 
![image](https://user-images.githubusercontent.com/11318903/64326391-33264400-cfca-11e9-82b3-cb5f236aff7e.png)